homework-submission/todo-app-week1/src/App.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react'
+import { BrowserRouter, Route, Link } from 'react-router-dom'
+import ItemsComponent from './components/ItemsComponent'
+import './App.css'
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      todos: new TodosModel(),
+    };
+    this.update = this.update.bind(this);
+
+  }
+
+  completeItem = id => {
+    let items = {
```

Comment:
this and the following lines has to be `const` :
 - https://github.com/HackYourFuture/React/pull/265/files#diff-00385634e950d968ea5a95be1c100b49R26
 - https://github.com/HackYourFuture/React/pull/265/files#diff-00385634e950d968ea5a95be1c100b49R33
 - https://github.com/HackYourFuture/React/pull/265/files#diff-840d62ec596eda126877dbe7e553604eR4
 - https://github.com/HackYourFuture/React/pull/265/files#diff-840d62ec596eda126877dbe7e553604eR5
 - https://github.com/HackYourFuture/React/pull/265/files#diff-840d62ec596eda126877dbe7e553604eR6